configs/server/forseti_conf_server.yaml.in

```diff
@@ -215,6 +209,15 @@ scanner:
 
 notifier:
 
+    # Provide connector details to receive violations when found
+    email_connector_config:
+          name: sendgrid
```

Comment:
The indenting here is inconsistent.  Please make it 2 spaces from the previous line